packages/eds-core-react/src/components/Table/Table.docs.mdx

```diff
@@ -96,6 +96,12 @@ Compact `Table` using `EdsProvider` (this will also apply compact on all nested
 
 ### Sortable
 
-This is just an example demonstrating, how simple sorting can be done. There are many different ways to solve it.
+This is just an example demonstrating how simple sorting can be done. There are many different ways to solve it.
 
-<Story id="data-display-table--sortable" />
\ No newline at end of file
+<Story id="data-display-table--sortable" />
+
+### Virtual scrolling
+Virtual scrolling is not supported in the EDS-table, and there are many different ways to implement it. Here 
+is an example of how it can be solved using <a href="https://tanstack.com/virtual/v3">react-virtual</a> 
```

Comment:
To render very large datasets, "virtual scrolling" might sometimes be a good option. EDS Table can be used in combination with 3rd party solutions such as <a href="https://tanstack.com/virtual/v3">react-virtual</a> to achieve this.